main.go

```diff
@@ -4,18 +4,22 @@ import (
 	"fmt"
 	"os"
 	"strconv"
+	"time"
+	"bytes"
 	"unicode"
 
 	"github.com/drone/drone-go/drone"
 	"github.com/drone/drone-go/plugin"
 )
 
-// HipChat represents the settings needed to send a HipChat notification.
-type HipChat struct {
-	Notify bool            `json:"notify"`
-	From   string          `json:"from"`
-	Room   drone.StringInt `json:"room_id_or_name"`
-	Token  string          `json:"auth_token"`
+// Options represents the settings from .drone.yml
+type Options struct {
```

Comment:
Does `HipChat` make more sense here?  I ask since there seemed to be some best practices/patterns/conventions emerging with drone plugins and this was one.  Here are examples from other plugins where this is the case and I just submitted a pull request for the webhook plugin that does the same. 

https://github.com/drone-plugins/drone-slack/blob/master/main.go#L22
https://github.com/drone-plugins/drone-tutum/blob/master/main.go#L12
https://github.com/drone-plugins/drone-npm/blob/master/main.go#L37
https://github.com/drone-plugins/drone-docker/blob/master/main.go#L42
https://github.com/drone-plugins/drone-webhook/pull/2/files#diff-7ddfb3e035b42cd70649cc33393fe32cR22

I wonder if there are any documents yet around these patterns... I know such a doc would help new contributors like myself :)